src/handler/commandHandler.ts

```diff
@@ -194,12 +194,38 @@ const commandMessageHandler = (
     message: Message,
     client: Client
 ): Promise<unknown> => {
+    const replaceLeet = (cmd: string): string => {
+        const alphabets: {[key: string]: string} = {
```

Comment:
a couple of substitutions can be made.
```
0 = O or D or space
1 = I or L
2 = Z
3 = E or e or m or w
4 = h or A or y
5 = S
6 = b or G
7 = T or j or L
8 = B or X
9 = g or J
```

Source: https://simple.wikipedia.org/wiki/Leet